parser_test.go

```diff
@@ -832,6 +832,45 @@ func TestParseRule(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "flowbits",
+			rule: `alert http $HOME_NET any -> $EXTERNAL_NET any (msg:"Flowbits test"; flow:to_server,established; content:"epaq"; http_uri; flowbits:set,Citrix_AGAX; flowbits:noalert; classtype:attempted-user; sid:2803611; rev:4;)`,
+			want: &Rule{
+				Action:   "alert",
+				Protocol: "http",
+				Source: Network{
+					Nets:  []string{"$HOME_NET"},
+					Ports: []string{"any"},
+				},
+				Destination: Network{
+					Nets:  []string{"$EXTERNAL_NET"},
+					Ports: []string{"any"},
+				},
+				SID:         2803611,
+				Revision:    4,
+				Description: "Flowbits test",
+				Tags: map[string]string{"flow": "to_server,established", "classtype": "attempted-user"},
```

Comment:
please add newlines between the `flow` and `classtype` entries for easier readability. Alternately remove these, since they're not necessary for the testing of flowbits parsing.